grossmont/computer-science-and-cybersecurity-organization/pages/about/about.html

```diff
@@ -100,12 +100,12 @@ <h4 style="text-align: center">Grossmont Computer Science and Cybersecurity Club
   </ul>
   <div style="padding: 30rem 0"></div>
   <div class="fixed-footer">
-    <iframe src="/grossmont/computer-science-and-cybersecurity-organization/sections/footer/footer.html" width="100%"
+    <iframe id='footer' src="/grossmont/computer-science-and-cybersecurity-organization/sections/footer/footer.html" width="100%"
       height="100%" frameborder="0"></iframe>
   </div>
   <script src="/jquery.min.js"></script>
-
   <script src="/footer-toggle.js"></script>
+  <script src="/theme.js"></script>
```

Comment:
If you want the dark light mode toggle to work on a certain page, you just need to assign the id of header and footer, and include the theme.js file after the jquery.min.js file.